### DISCUSSION TITLE
fa/manager/installation-and-setup/Quick-Installation-on-Microsoft-Azure/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4356] [Content Help کمک به اصلاح محتوا]
# fa/manager/installation-and-setup/Quick-Installation-on-Microsoft-Azure/

The proxy manager platform.

https://hiddify.com/fa/manager/installation-and-setup/Quick-Installation-on-Microsoft-Azure/

**Comment by Johan-Mohan** [comment]
deploy to azure icon doesnt work properly 
contents from github data centre are removed
